### PR TITLE
Cursory glance of Introduction section

### PR DESCRIPTION
**Pull request by ngscheurich**
This makes some changes to text of the book—mostly objective, some shaded by my stylistic preferences. No code or code examples were harmed in the making of this pull request.

### CODE REVIEW

File: book/introduction/tooling.md [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
 
-What is Ncurses you ask? Ncurses stands for New Curses, it's a freeware reimplementation of the original curses library. Its purpose is to make managing screen state easier and more portable. If you've ever installed some flavor of linux on a computer you might have seen something like this:
+What is ncurses you ask? ncurses stands for *new curses*, it's a freeware reimplementation of the original curses library. Its purpose is to make managing screen state easier and more portable. If you've ever installed some flavor of Linux on a computer you might have seen something like this:
 
 \includegraphics[width=\linewidth]{images/ncurses-example.png}
 
-This screen was created using Ncurses. So why do **we** need Ncurses? After all, we're just writing a simple game. Using Ruby let's try to do something simple like clearing the screen. If you're on OSX you might have written something like:
+This screen was created using Ncurses. So why do *we* need Ncurses? After all, we're just writing a simple game. Using Ruby let's try to do something simple like clearing the screen. If you're on macOS you might have written something like:
```
Comment (richdouglasevans) [inline book/introduction/tooling.md]: The references to "Ncurses" want to be changed to "ncurses" for consistency, no?

File: book/introduction/what_is_a_roguelike.md [hunk 2]
```diff
@@ -13,12 +13,12 @@ So where do these rules come from? Roguelike games get their rules and name from
...
 
-An important part of a roguelike game is having to think very hard about your next move. Quick! You're about to die! Do you pray, do you write "Elbereth" on the ground hopping the never ending assult of ants will leave your pitiful tourist alone! Each keypress could be your last. For this reason most roguelikes are turn-based. As the player, you'll issue a single command and the entire world will update. This gives you a chance to examine all the changes and carefully plan how you'll deal with the situation at hand.
+An important part of a roguelike game is having to think very hard about your next move. Quick! You're about to die! Do you pray, do you write "Elbereth" on the ground hopping the never ending assult of ants will leave your pitiful tourist alone? Each keypress could be your last. For this reason most roguelikes are turn-based. As the player, you'll issue a single command and the entire world will update. This gives you a chance to examine all the changes and carefully plan how you'll deal with the situation at hand.
```
Comment (richdouglasevans) [inline book/introduction/what_is_a_roguelike.md]: Some typos:

* "never ending" :arrow_right: "never-ending" ([Why?](http://inventingrealityeditingservice.typepad.com/inventing_reality_editing/2016/01/never-ending-vs-neverending-vs-never-ending.html))
* "assult" :arrow_right: "assault"

File: book/introduction/what_is_a_roguelike.md [hunk 3]
```diff
@@ -13,12 +13,12 @@ So where do these rules come from? Roguelike games get their rules and name from
...
 
-Which brings us to the next rule... permanent death! This is the crux of a roguelike. If you die, you lose EVERYTHING and have to start over. All your hard work! That amazing wand you found! All gone! This means your choices have serious consequences. This makes success very rewarding and failure very frustrating. Do you read a scroll that could destroy your armor making you defenseless, but also could detect food you so desperately need? This tension is rarely found in games of other ilk.
+Which brings us to the next rule... permanent death! This is the crux of a roguelike. If you die, you lose *everything* and have to start over. All your hard work! That amazing wand you found! All gone! This means your choices have serious consequences. This makes success very rewarding and failure very frustrating. Do you read a scroll that could destroy your armor making you defenseless, but also could detect food you so desperately need? This tension is rarely found in games of other ilk.
```
Comment (richdouglasevans) [inline book/introduction/what_is_a_roguelike.md]: Do we want to replace synthetic ellipses (...) with actual ellipses? (…)

File: book/introduction/tooling.md (hunk 1 above)
Comment (ngscheurich) [reply book/introduction/tooling.md]: Good catch. Fixed these and one more occurrence in 80e682f7729c9e7b25a0d44d1594d244c7d9042b.

File: book/introduction/what_is_a_roguelike.md (hunk 2 above)
Comment (ngscheurich) [reply book/introduction/what_is_a_roguelike.md]: Perfect. Thanks for the reference link re: *never-ending*.

File: book/introduction/what_is_a_roguelike.md (hunk 3 above)
Comment (ngscheurich) [reply book/introduction/what_is_a_roguelike.md]: Yes, absolutely! I've also been thinking about replacing straight quotes with curly quotes—should we, and how? Maybe we can hook into a build step to do these sorts of conversions?

Comment (ngscheurich) [reply book/introduction/what_is_a_roguelike.md]: Check out #31.

**Pull request merged**